### DISCUSSION TITLE
Quorum queues acceptible with one node

### DISCUSSION THREAD

**Discussion by jimmycartrette** [rabbitmq/rabbitmq-server#10876] [Questions] [answered]
We're looking at switching our application over to use quorum queues where applicable from mirrored queues. We will handle the backend creation of the queues. My question is around setting quorum queues as the default vs having them select to use quorum queues if they have more than 2 nodes. The text from the documentation here:

Therefore the recommended number of replicas for a quorum queue is the quorum of cluster nodes (but no fewer than three)

As I've been testing our code, I have a one node setup (which is probably like the majority of our customers) as it creates and uses quorum queues. What is the downside to using it with one node? If there were no downsides we'd just have it create quorum queues over classic queues as the default for new customers.

**Comment by michaelklishin** [comment, marked as answer]
With just one node you will miss on the key benefits of quorum queues, namely the replication aspect. Otherwise, using a QQ in a single node cluster at first is fine.

The biggest downside compared to a non-mirrored classic queue v2 is that you can have fewer QQs per node as they consume more resources when idle. But you can still have a few thousands without any tweaking.
